app/routes.py

```diff
@@ -1,6 +1,7 @@
-from flask import render_template, jsonify, request
+from flask import flash, render_template, jsonify, request
```

Comment:
flash lets you forward a message to the next request, it looks like. I believe that's what lets the search entry create the results beneath. I'm considering changing it up so that isn't needed, but it might be needed.